### ISSUE TITLE
Java 8 is not supported

### ISSUE THREAD

**Issue by guangwang126** [objectionary/ineo-maven-plugin#73]
Does 'opeo-maven-plugin' have any plans to support Java 8?

**Comment by guangwang126** [comment]
@volodya-lombrozo Can you try to help?

**Comment by volodya-lombrozo** [comment]
@guangwang126 Yes, of course. I'm working on it https://github.com/objectionary/ineo-maven-plugin/pull/72

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon Could you release a new version with Java 8 support? https://github.com/objectionary/ineo-maven-plugin/pull/72

**Comment by maxonfjvipon** [comment]
@rultor release, tag is `0.3.2`

**Comment by maxonfjvipon** [comment]
@rultor release, tag is `0.3.2`

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon Maven are changing authentication mechanisms these days:

> We are making changes to the OSSRH authentication backend. For most users this should be a transparent process, and you should be able to continue to use your existing username and password to connect the Nexus UI. In case you need to update your password, please [follow our documentation](https://central.sonatype.org/register/central-portal/#managing-your-credentials).
To configure a publisher’s plugin authentication you would need to update your plugin settings to use [a user token](https://central.sonatype.org/publish/generate-token/) instead of the Nexus UI username and password login.
For more information about publishing to legacy OSSRH please consult our documentation at [https://central.sonatype.org/register/legacy/](https://central.sonatype.org/register/legacy/) 

So, most probably, we need to fix authentication somehow. @yegor256 Could you help us, please?

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon Reminder. I'm still need this.

**Comment by yegor256** [comment]
@maxonfjvipon @volodya-lombrozo should be fixed now, try again

**Comment by maxonfjvipon** [comment]
@rultor release, tag is `0.3.2`

**Comment by yegor256** [comment]
@maxonfjvipon it's weird. I set the token as required by the their doc. Let me check again.

**Comment by yegor256** [comment]
@rultor release, tag is `0.3.2`

**Comment by maxonfjvipon** [comment]
@rultor release, tag is `0.3.2`

**Comment by volodya-lombrozo** [comment]
@yegor256 Seems, we have a different exception now:

```java
Server credentials with ID "oss.sonatype.org" not found!
```

Could you check server credentials, please?

**Comment by volodya-lombrozo** [comment]
@yegor256 Friendly reminder.

**Comment by yegor256** [comment]
@rultor release, tag is `0.3.2`

**Comment by volodya-lombrozo** [comment]
@maxonfjvipon Friendly reminder.

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you take a [look](https://github.com/objectionary/ineo-maven-plugin/issues/73#issuecomment-2186301074), please?

**Comment by yegor256** [comment]
@volodya-lombrozo I don't know what to do... the token is correct, configured properly. Why they don't accept it - no idea. I will email them now.

**Comment by yegor256** [comment]
@volodya-lombrozo looks like we are not alone: https://community.sonatype.com/t/401-content-access-is-protected-by-token-authentication-failure-while-performing-maven-release/12741

**Comment by yegor256** [comment]
@rultor release, tag is `0.3.2`

**Comment by maxonfjvipon** [comment]
@rultor release, tag is `0.3.2`

**Comment by yegor256** [comment]
@volodya-lombrozo @maxonfjvipon fixed

**Comment by volodya-lombrozo** [comment]
@guangwang126 now `ineo-maven-plugin` supports Java 8 as well.
